### PR TITLE
PRO-1176: PLR Staking UI Changes

### PR DESCRIPTION
**Pull request by Yash-Chaubal**
https://linear.app/pillarproject/issue/PRO-1176/pillar-wallet-plr-staking

## Description

- Add banner to top of home screen for PLR Staking
- Button in investments section now displays remaining time
- Include PLR Staked token in the header of the Staking home screen
- Couldn't find out how to get ETH value of rewards, so showed fiat instead (staked PLR * apy)

## Screenshots
Simulator Screenshot - iPhone 14 Pro - 2023-08-16 at 13 37 57
Simulator Screenshot - iPhone 14 Pro - 2023-08-16 at 13 38 02

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by IAmKio** [comment]
@Yash-Chaubal is this still a WIP?

File: src/screens/PlrStaking/PlrStaking.tsx [hunk 1]
```diff
@@ -180,6 +191,22 @@ const PlrStaking = () => {
...
 
+  useEffect(() => {
+    const fetchBal = async () => {
+      let totalStkPlr = BigNumber.from(0);
+      for (let account in accounts) {
+        const accountAddress = getAccountAddress(account);
+        const balance = await getBalanceForAddress(CHAIN.ETHEREUM, stakedTokenAddress, accountAddress);
+        if (balance && !balance.isZero()) totalStkPlr.add(balance);
+      }
+      console.log({ totalStkPlr });
```
Comment (parag-chauhan) [inline src/screens/PlrStaking/PlrStaking.tsx:202]: Can we remove this console.log.

File: src/screens/PlrStaking/PlrStaking.tsx [hunk 2]
```diff
@@ -286,6 +310,28 @@ const PlrStaking = () => {
         stakedPercentage={stakedPercentage}
         stakers={stakers}
         apy={stakingApy}
+        renderStakedInfo={() => {
```
Comment (parag-chauhan) [inline src/screens/PlrStaking/PlrStaking.tsx:313]: can we create a seperate function like `stakedInfoWrapper` or something like that and use here.

**Review by parag-chauhan — APPROVED** [review]
Just couple of comments. LGTM @Yash-Chaubal

**Pull request merged**